pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
@lucleray Looks like you forgot to run `corepack enable pnpm` on your machine because this was likely using a different version of pnpm.

Reply:
I think I did 🤔 

I'll double check though.

Reply:
Ok, I think you were right. I've enabled corepack and I can "cleanly" install without altering the pnpm-lock.yaml file.

That said, I don't think these added lines caused any trouble for the examples.

Reply:
I also noticed something else: when cloning the examples, the pnpm version used currently is `8.11.0` (not `8.3.1` like defined in the root package.json).

(That works as expected, because only the subfolder is cloned)

We should probably upgrade the pnpm version of the repo 🤔